src/lib/sidenav/sidenav.ts

```diff
@@ -271,11 +274,16 @@ export class MdSidenavLayout implements AfterContentInit {
   /** TODO: internal */
   ngAfterContentInit() {
     // On changes, assert on consistency.
-    this._sidenavs.changes.subscribe(() => this._validateDrawers());
+    this._sidenavChangesSubscription = this._sidenavs.changes
```

Comment:
This one also shouldn't be necessary because the `QueryList` is destroyed when the component is destroyed